diff --git a/eek/layersurface.c b/../../squeekboard-1.43.1/eek/layersurface.c
index d62cc70..56f5775 100644
--- a/eek/layersurface.c
+++ b/../../squeekboard-1.43.1/eek/layersurface.c
@@ -769,6 +769,12 @@ phosh_layer_surface_set_layer (PhoshLayerSurface *self, guint32 layer)
   g_return_if_fail (PHOSH_IS_LAYER_SURFACE (self));
   priv = phosh_layer_surface_get_instance_private (self);
 
+  /* Check env var override */
+  //const char *env = g_getenv ("SQUEEKBOARD_LAYER");
+  //if (env && g_strcmp0 (env, "overlay") == 0) {
+    layer = 3; // ZWLR_LAYER_SHELL_V1_LAYER_OVERLAY
+  //}
+
   if (priv->layer == layer) {
     return;
   }
